### PR TITLE
Indexed triangle strip cut for TriangleLL

### PR DESCRIPTION
**Pull request by kvark**
This change was done for a GPUWeb test (positive!), but I don't think why we'd want this wasted ;)

### CODE REVIEW

File: src/backend/dx12ll/src/factory.rs [hunk 1]
```diff
@@ -421,7 +421,8 @@ impl core::Factory<R> for Factory {
                     pInputElementDescs: input_element_descs.as_ptr(),
                     NumElements: input_element_descs.len() as u32,
                 },
-                IBStripCutValue: winapi::D3D12_INDEX_BUFFER_STRIP_CUT_VALUE_DISABLED,
+                //IBStripCutValue: winapi::D3D12_INDEX_BUFFER_STRIP_CUT_VALUE_DISABLED,
+                IBStripCutValue: winapi::D3D12_INDEX_BUFFER_STRIP_CUT_VALUE_0xFFFFFFFF, //note: this works with U16 too!
```
Comment (msiglreith) [inline src/backend/dx12ll/src/factory.rs]: Looks a bit hackish, but if it works ..

**Comment by kvark** [comment]
Oh great:
> error: An unknown error occurred

**Comment by msiglreith** [comment]
Travis also fails with unknown errors, let's try again ..

bors r+

**Comment by msiglreith** [comment]
Travis passed in the 2nd run and appveyor only fails on nightly, probably some temporary regression.
Merging it manually.

**Pull request merged**